front-end/src/components/Teacher/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+// FontAwesome library
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebookF,
+  faTwitter,
+  faInstagram,
+  faLinkedinIn,
+} from '@fortawesome/free-brands-svg-icons';
+import { faShareNodes } from '@fortawesome/free-solid-svg-icons';
+
+// Components
+import Tabs from 'components/common/Tabs';
+import Overview from 'components/Teacher/Overview';
+import Courses from 'components/Teacher/Courses';
+
+// Styles
+import styles from './Teacher.module.scss';
+
+const Teacher = ({ fullName, activity, id, overview, courses }) => {
+  const { pathname } = useLocation();
+  const tabs = [
+    { title: 'Overview', link: `/app/teachers/${id}/overview` },
+    { title: 'Courses', link: `/app/teachers/${id}/courses` },
+  ];
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.test}>
+        <div className={styles.iconsWrap}>
+          <FontAwesomeIcon icon={faFacebookF} />
+          <FontAwesomeIcon icon={faTwitter} />
+          <FontAwesomeIcon icon={faInstagram} />
+          <FontAwesomeIcon icon={faLinkedinIn} />
+        </div>
+        <div className={styles.imageWrap}>
+          <img
+            src='https://storage.googleapis.com/teachers-platform-40cbe.appspot.com/f53bdc08d852eafe73d86a7c78847814.jpg'
+            alt='teacher'
+          />
+        </div>
+        <div className={styles.share}>
+          <FontAwesomeIcon icon={faShareNodes} />
+          <span>Report This Author</span>
+        </div>
+      </div>
+      <div className={styles.description}>
+        <h1>{fullName}</h1>
+        <span>{activity}</span>
+      </div>
+      <div className={styles.test1}>
+        <div className={styles.blockWrap}>
+          <img src='https://storage.googleapis.com/teachers-platform-40cbe.appspot.com/e5b18e70845a38cb3232b3625f2f40f9.png' />
```

Comment:
Please create constants for photos which were upload on cloud and then use as variable in code

Reply:
fix it

Reply:
fix it